roles/tests-integration/tests/common/mod.rs

```diff
@@ -282,3 +282,101 @@ pub async fn start_template_provider(tp_port: u16) -> TemplateProvider {
     template_provider.generate_blocks(16);
     template_provider
 }
+
+pub async fn start_jdc(
+    pool_address: SocketAddr,
+    tp_address: SocketAddr,
+    jds_address: SocketAddr,
+) -> SocketAddr {
```

Comment:
please see #1234

Reply:
This is initializing the `get_available_address` internal and returns it here, which is I think what we want